### DISCUSSION TITLE
1x1 widget just to open Quicknote

### DISCUSSION THREAD

**Discussion by Yellow-Subroutine** [gsantner/markor#2010] [Ideas] [answered]
Hi,

I find the current widget too large and not all that useful for what I do, all I want is a way to quickly open the Quicknote-document.

I have made myself a button to do that with tasker but it would be an improvement if markor would already provide such a widget.

Ufortunately I don't know how to implement widgets in android, so all I can offer is the request for it...

**Comment by gsantner** [comment, marked as answer]
There is a settings option "Special Launchers". It gives you a launcher specifically to open QuickNote. Should work for what you want.
